Dockerfile

```diff
@@ -136,9 +150,5 @@ RUN mkdir -p "${AIRFLOW_DAG}"
 
 COPY setup_auth.py test_db_conn.py ${AIRFLOW_HOME}/
 
-# For S3 logging feature
-COPY ./config/ "${AIRFLOW_HOME}/config/"
-
 # All the other env vars that don't affect the build here
-ENV PYTHONPATH="${PYTHONPATH}:${AIRFLOW_HOME}/config"
```

Comment:
The config template that uses `AIRFLOW__LOGGING__REMOTE_BASE_LOG_FOLDER`, guess it doesn't need anything for it to just work?

Reply:
This step seems only necessary if we want to override logging values that are not exposed via env vars or `airflow.cfg` for both 1.10.* and 2.1.*:

https://airflow.apache.org/docs/apache-airflow/stable/logging-monitoring/logging-tasks.html#advanced-configuration
https://airflow.apache.org/docs/apache-airflow/1.10.12/howto/write-logs.html#enabling-remote-logging